apps/web/components/booking/TimeOptions.tsx

```diff
@@ -27,13 +27,11 @@ const TimeOptions: FC<Props> = (props) => {
     if (selectedTimeZone && timeZone() && selectedTimeZone !== timeZone()) {
       props.onSelectTimeZone(timeZone(selectedTimeZone));
     }
-  }, [selectedTimeZone]);
-
+  }, [selectedTimeZone, props.onSelectTimeZone]);
```

Comment:
Added this to remove the warning, but maybe we should properly deconstruct onSelectTimeZone as I am not sure if this makes it rerender always that any props change.

Reply:
Yup actually merging back main eith new eslint config bite my in the asss with this one heh